apps/sms/test/marionette/lib/views/conversation/accessors.js

```diff
@@ -110,6 +110,16 @@ ConversationAccessor.prototype = {
     this.actions.flick(conversationContainer, 50, 50, 50, 350).perform();
   },
 
+  fakeScrollUpTo: function(scrollTop) {
+    var conversationContainer = this.client.findElement(
+      SELECTORS.container
+    );
+
+    conversationContainer.scriptWith(function(container, scrollTop) {
+      container.scrollTop = scrollTop;
+    }, scrollTop);
+  },
```

Comment:
mm, imo, it's too low level, what is wrong with ```scrollUp``` function? I remember you had problem with displayness but don't remember what exactly was the problem. Maybe we should just improve it if it's not useful enough?